scripts/scil_transform_surface.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to load and transform cortical surface (vtk or freesurfer),
+This script is using ANTs transform (affine.txt, warp.nii.gz).
+
+Best usage for with ANTs from T1 to b0:
+> ConvertTransformFile 3 output0GenericAffine.mat vtk_transfo.txt --hm
+> scil_transform_surface.py lh_white_lps.vtk affine.txt lh_white_b0.vtk\\
```

Comment:
From the filename I would think for that script to work "as intended" the input surface should be aligned with the T1 in MI-Brain and then the output would be aligned with the B0 in MI-Brain after.
Could you mention explicitly that the surface need to be Ips, or even simpler mention the MI-Brain sanity check so the user know in which 'space' the input must be.